packages/documentation/src/stories/accessibility-practices/foundational-structure-and-semantics/reference-crossing-the-shadowdom/aria-describedby/aria-describedby.docs.mdx

```diff
@@ -0,0 +1,171 @@
+import DefaultExample from './examples/defaultExample.sample.html?raw';
+import DefaultExample2 from './examples/defaultExample2.sample.html?raw';
+import AriaDescribedbyElementsPropertyHTML from './examples/ariadescribedbyElementsProperty.sample.html?raw';
+import AriaDescribedbyElementsProperty from './examples/ariadescribedbyElementsProperty.sample.js?raw';
+import NonWorkingExampleIIa from './examples/nonWorkingExample-IIa.sample.html?raw';
+import NonWorkingExampleIIb from './examples/nonWorkingExample-IIb.sample.html?raw';
+import WorkingExampleIIa from './examples/workingExample-IIa.sample.html?raw';
+import WorkingExampleIIb from './examples/workingExample-IIb.sample.html?raw';
+import NonWorkingExampleIII from './examples/nonWorkingExample-III.sample.html?raw';
+import WorkingExampleIII from './examples/workingExample-III.sample.html?raw';
+import NonWorkingExampleIV from './examples/nonWorkingExample-IV.sample.html?raw';
+import WorkingExampleIV from './examples/workingExample-IV.sample.html?raw';
+
+<Meta of={CCR} />
+
+# aria-describedby
+
+<div className="lead">
+  In the Light DOM, `aria-describedby` is used to give an element a description by referencing the
+  ID of another element or multiple elements that contain the desired text.
+</div>
+<p>
+  This is useful when a simple text label isn't enough or when the description needs to be combined
+  from different pieces of text.
+</p>
+
+### I. Referencing Within the Same DOM Tree
+
+• <b>Both elements in Light DOM</b> ✔️
+
+<Source language="html" dark code={DefaultExample}></Source>
+
+• <b>Both elements in same Shadow DOM</b> ✔️
+
+<Source language="html" dark code={DefaultExample2}></Source>
+<p>
+  <b>
+    The above relationships are valid because in both cases the target element and its description
+    are part of the same tree.
+  </b>
+</p>
+
+<p>
+  Below is a working example of an `aria-describedby`/`id` relationship within the shadowDom of a
+  custom button component. You can <b>test it with NVDA</b> to confirm the correct description
+  announcement.
+</p>
+<Canvas of={CCR.ExampleHTML} sourceState="hide" />
+
+
+### II. Referencing From Outside a Shadow DOM Into That Shadow DOM
+
+• <b>Light DOM → Shadow DOM</b> ❌
+
+<Source language="html" dark code={NonWorkingExampleIIa}></Source>
+
+• <b>Slotted Content → Shadow DOM</b> ❌
+
+<Source language="html" dark code={NonWorkingExampleIIb}></Source>
+
+<p>
+  The above examples do not work across shadow DOM boundaries when setting the aria-describedby
+  attribute directly.
+</p>
+<p>
+  <b>
+    Attribute element references can only point to elements in the same DOM or shadow DOM, because
+    IDs are only valid within their own scope.
+  </b>
+</p>
+
+#### Workaround: `Element:ariaDescribedByElements`
+
+<p>
+  `ariaDescribedByElements` is a property of the Element interface that returns an array of elements
+  describing the element. It provides direct references to these elements rather than just their ID
+  strings.
```

Comment:
It could be helpful for some DEVs, if you add the information, that this is a [WebAPI](https://developer.mozilla.org/en-US/docs/Web/API) and therefore JavaScript related.